en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -784,6 +784,11 @@ The N + 1 query problem can be alleviated using eager loading which in this exam
 $teams = Team::get()->eagerLoad('Players');
 ```
 
+> [!CAUTION]
+> Filtering an `EagerLoadedlist` is significantly (up to 85%) slower than performing that manipulation on the eager loading query.
+>
+> See [manipulating eagerloading queries](#manipulating-eager-loading-queries).
```

Comment:
Added this here as an early warning about this, for people who don't read the whole docs (i.e. most everyone).
It is intentionally duplicating a warning in that section.